src/routes/LearningRoute/LearningRoute.js

```diff
@@ -48,6 +48,7 @@ class LearningRoute extends Component {
         newCurrentWord.answer = res.answer
         if(res.isCorrect){
           newCurrentWord.wordCorrectCount++;
+          newCurrentWord.totalScore++;
```

Comment:
the score was being updated in the server, so rather than make an api call, i just told the client to assume score is incremented